spec/asyncapi.md

```diff
@@ -399,128 +398,82 @@ protocol: kafka
 protocolVersion: '1.0.0'
 ```
 
-The following shows how multiple servers can be described, for example, at the AsyncAPI Object's [`servers`](#A2SServers):
+Using server variables for dynamic URLs:
 
 ```json
 {
-  "servers": {
-    "development": {
-      "url": "development.gigantic-server.com",
-      "description": "Development server",
-      "protocol": "amqp",
-      "protocolVersion": "0.9.1",
-      "tags": [
-        { 
-          "name": "env:development",
-          "description": "This environment is meant for developers to run their own tests"
-        }
-      ]
+  "url": "{username}.gigantic-server.com:{port}/{basePath}",
```

Comment:
Here I'm just removing context information in the example. Just to keep it the same as in the rest of the spec.